src/components/bulletin-components/BulletinMessage.tsx

```diff
@@ -5,7 +5,7 @@ type BulletinMessageProps = {
 };
 
 type BulletinMessageState = {
-  expanded: boolean;
+  displayMessage: string;
```

Comment:
You don't need to use state in your case. Since you are creating shorter string based on the props :)

So something like this in render
![image](https://user-images.githubusercontent.com/12229968/63962460-97fa0f80-ca9b-11e9-867d-b25e2e6cc416.png)